### PR TITLE
yarn: add page

### PR DESCRIPTION
**Pull request by vkolmakov**
Pretty much the npm page translated to [yarn](https://yarnpkg.com/) with few changes/additions.

### CODE REVIEW

**Comment by waldyrious** [comment]
Thanks for the contribution! Could you reduce it a bit, though? [Our guidelines](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines) recommend sticking to at most 5-6 examples per page.

**Comment by vkolmakov** [comment]
Sure, I can merge add and add-dev, and remove ls.

File: pages/common/yarn.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# yarn
+
+> JavaScript and Node.js package manager alternative.
+
+- Install a module globally:
+
+`yarn global add {{module_name}}`
+
+- Install all dependencies referenced in the package.json:
```
Comment (waldyrious) [inline pages/common/yarn.md]: maybe add "file" at the end of the sentence.

File: pages/common/yarn.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+`yarn`
+
+- Install a module and save it as a dependency to the package.json. Add --dev to save as a dev dependency:
+
+`yarn add {{module_name}}@{{version}}`
+
+- Uninstall a module and remove it from the package.json:
```
Comment (waldyrious) [inline pages/common/yarn.md]: same here about the "the".

File: pages/common/yarn.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+`yarn remove {{module_name}}`
+
+- Interactively create a package.json file:
+
+`yarn init`
+
+- Identify why the module is installed and list other modules that depend on it:
```
Comment (waldyrious) [inline pages/common/yarn.md]: "why" is a little vague. What kind of justifications does this subcommand provide?

**Comment by vkolmakov** [comment]
@waldyrious sounds good, thanks for the feedback!

**Comment by agnivade** [comment]
LGTM

**Comment by waldyrious** [comment]
LGTM as well 👍 thanks for the quick reactions!

**Pull request merged**